manager/controllers/app/plotter_controller.go

```diff
@@ -297,7 +300,23 @@ func NewPlotterReconciler(mgr ctrl.Manager, name string, manager multicluster.Cl
 
 // SetupWithManager registers Plotter controller
 func (r *PlotterReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	// 'UpdateFunc' and 'CreateFunc' used to judge if the event came from within the system namespace.
+	// If that is true, the event will be processed by the reconciler.
+	// If it's not then it is a rogue event created by someone outside of the control plane.
```

Comment:
I think rogue event is a mit misleading. Those events were not created by someone. Those events come from the creation/update/deletion of plotter objects. Those objects just happen to be in the wrong namespace and are thus ignored by the controller. It might be a good idea to log these events being filtered out. So that if someone expects the objects to progress but they don't because they are in the wrong namespace it's mentioned in the logs that the plotter is in the wrong namespace and is thus ignored.

Reply:
I think the point is to minimize any work being done. Doesn't it include logs? I don't think a plotter would be created manually "by mistake".

Reply:
Ok. I'm ok with that. A plotter is an internal object.